### PR TITLE
Add //deleteblocks command

### PR DESCRIPTION
**Pull request by est31**
same as #71 , had to close/open it for a technical reason.

### CODE REVIEW

**Comment by Uberi** [comment]
:+1:

**Comment by est31** [comment]
Before merge, I want to update doc to point  out that mapgen can change more than the current block.

File: ChatCommands.md [hunk 1]
```diff
@@ -98,6 +98,12 @@ Display the volume of the current WorldEdit region.
 
     //volume
 
+### `//deleteblocks`
+
+Delete the mapblocks (16x16x16 units) that contain the selected region. This means that mapgen will be invoked for that area. As only whole mapblocks get removed, the deleted area is usually larger than the selected one.
```
Comment (sfan5) [inline ChatCommands.md]: minor nitpick: it's `MapBlock`

File: worldedit_commands/init.lua [hunk 2]
```diff
@@ -277,6 +277,21 @@ minetest.register_chatcommand("/volume", {
 	end,
 })
 
+minetest.register_chatcommand("/deleteblocks", {
+	params = "",
+	description = "remove all mapblocks (16x16x16) containing the selected area from the map",
```
Comment (sfan5) [inline worldedit_commands/init.lua]: same

**Comment by ShadowNinja** [comment]
@est31 The maximum affected size is probably a chunk -- which is 80x80x80 by default, so running the command on a 2x2x2 area could affect up to a 160x160x160 area.

**Comment by est31** [comment]
Made mapblocks CamelCase, and updated doc, this is ready for merge.

**Comment by Uberi** [comment]
Looks good!

**Pull request merged**